### PR TITLE
[refactor] Resolve issues on LeXimir branch after PR #130

### PR DESCRIPTION
**Pull request by FLambertD**
This is the application of the review recommendations of the [PR#130](https://github.com/UnitexGramLab/gramlab-ide/pull/130).
It is mostly conventions applications like changing variable names or removing useless spaces and blanks.

## Motivation and Context
Context constrained to merge the PR#130 without apply the review changes. This PR is fixing it.

## How Has This Been Tested?
No tests were needed because this is a simple refactor.

## Type of files

- [ ] `bin`: Binary files
- [ ] `ci`: Continuous integration files
- [ ] `doc`: Documentation files
- [x] Plain-text source code files

## Level of change

- [ ] `break`: Breaking change
- [ ] `exp`: Experimental change
- [ ] `tmp`: Temporal change
- [ ] `major`: Major change
- [x] `minor`: Minor change
- [ ] `sec`: Vulnerability-related change
- [ ] None of the above (normal change)

## Type of change

- [ ] `deprecat`: Deprecation of a once-stable feature
- [ ] `enhance`: Enhancement in existing functionality
- [ ] `fix`: Bug fix
- [ ] `feature`: New feature
- [ ] `hotfix`: Hotfix for bugs
- [x] `refactor`: Improve coding style, comments
- [ ] `remove`: Remove a feature

## Checklist:

- [x] My code compiles
- [x] My code follows the code style of this project
- [x] My code includes javadoc/doxygen where appropriate
- [x] My code is well factored, so that there is not repetitive code in the wild
- [x] My code does not require a change in the documentation, if so I already opened an issue to list the changes
- [x] I have read the **CONTRIBUTING** document
- [x] I have read the **Commit Message Guidelines**
- [x] I understand that all commits on my pull request will be squashed to a single good one

### CODE REVIEW

File: unitex/src/fr/umlv/unitex/leximir/helper/DelasHelper.java [hunk 1]
```diff
@@ -80,17 +80,21 @@ else if(dic!=null){
         for(String dela:list){
             //String path = Utils.getValueXml("pathDelas")+"/"+dela;
             String path="";
-            if(allDelas)
-            	path = DictionaryPath.allDelas+File.separator+dela;
-            else
+            if(allDelas){
+            	path = DictionaryPath.allDelas+File.separator + dela;
```
Comment (eric-laporte) [inline unitex/src/fr/umlv/unitex/leximir/helper/DelasHelper.java:84]: All binary operators except `.` should be separated from their operands by spaces

File: unitex/src/fr/umlv/unitex/config/UserPreferences.java [hunk 2]
```diff
@@ -152,7 +152,7 @@ public boolean setRecentDelas(String link) {
 			return false;
 		}
 		return true;
-	}                   
+	}
```
Comment (eric-laporte) [inline unitex/src/fr/umlv/unitex/config/UserPreferences.java:155]: Make sure to remove trailing whitespace

File: unitex/src/fr/umlv/unitex/leximir/helper/StatisticOutput.java [hunk 3]
```diff
@@ -145,7 +142,7 @@ private void jButtonSetActionPerformed(java.awt.event.ActionEvent evt) {
         if (result == JFileChooser.APPROVE_OPTION) {
 	        File selectedFile = fileChooser.getSelectedFile();
 	        EditorDelas.setStatisticDirectory(selectedFile.getAbsolutePath()+File.separator);
-	        jTextField1.setText(EditorDelas.getStatisticDirectory()+"statisticsTmp.csv");
+	        jTextField1.setText(EditorDelas.getStatisticDirectory()+DictionaryPath.statisticsTmpPath);
```
Comment (eric-laporte) [inline unitex/src/fr/umlv/unitex/leximir/helper/StatisticOutput.java:145]: All binary operators except `.` should be separated from their operands by spaces

**Review by eric-laporte — APPROVED** [review]
I approve the changes and I suggest a few more changes on details

File: unitex/src/fr/umlv/unitex/config/UserPreferences.java (hunk 2 above)
Comment (FLambertD) [reply unitex/src/fr/umlv/unitex/config/UserPreferences.java:155]: I'm not sure to understand, i removed the whitespace in this example.

Comment (eric-laporte) [reply unitex/src/fr/umlv/unitex/config/UserPreferences.java:155]: When I download UserPreferences.java, I see 15 spaces in the end of line 155. Also in lines 99 140 148...

Comment (FLambertD) [reply unitex/src/fr/umlv/unitex/config/UserPreferences.java:155]: Right, I only removed tab.

**Review by eric-laporte — APPROVED** [review]

File: unitex/src/fr/umlv/unitex/leximir/helper/StatisticOutput.java [hunk 4]
```diff
@@ -152,14 +152,17 @@ private void jButtonSaveActionPerformed(java.awt.event.ActionEvent evt) {
         	String filename= jTextField1.getText();
         	File saveFolder = new File(new File(filename).getParent());
         	if(!saveFolder.exists()) {
-        		 int reply = JOptionPane.showConfirmDialog(null, "This folder doesn't exist yet \nCreate this folder?", title,
+        		 int reply = JOptionPane.showConfirmDialog(null, "This folder \""+saveFolder.getName() +"\" doesn't exist yet \nCreate this folder?", title,
                          JOptionPane.YES_NO_OPTION);
                  if (reply == JOptionPane.YES_OPTION) {
                      saveFolder.mkdirs();
                      if (saveFolder.exists()) {
                          JOptionPane.showMessageDialog(null, "The folder created successfully !", "Information", JOptionPane.INFORMATION_MESSAGE);
```
Comment (eric-laporte) [inline unitex/src/fr/umlv/unitex/leximir/helper/StatisticOutput.java:160]: There should be no space before "!" in English

File: unitex/src/fr/umlv/unitex/leximir/util/DuplicationFinder.java [hunk 5]
```diff
@@ -128,7 +128,7 @@ protected void done() {
         if(this.jtableRes.getModel().getRowCount()>0) {
             new MenuDuplicate(this.jtableRes);
         } else {
-            JOptionPane.showMessageDialog(null, "No duplication where found !", "Duplication", JOptionPane.INFORMATION_MESSAGE);
+            JOptionPane.showMessageDialog(null, "No duplication found !", "Duplication", JOptionPane.INFORMATION_MESSAGE);
```
Comment (eric-laporte) [inline unitex/src/fr/umlv/unitex/leximir/util/DuplicationFinder.java:131]: There should be no space before '!' in English

**Review by eric-laporte — COMMENTED** [review]
I approve the changes, with two requests on messages

**Review by eric-laporte — APPROVED** [review]

**Pull request merged**